german-quinonez/lesson-1/homework-2-Car/src/Car.java

```diff
@@ -0,0 +1,73 @@
+public class Car {
+
+    private double gas;
+    private double distance;
+    private double gasConsume;
+    private boolean isPowerOn;
+    private double capacity;
+
+
+    public Car() {
+        this.gas = 0.0;
+        this.distance = 0.0;
+        this.gasConsume = 0.0;
+        this.isPowerOn = false;
+        this.capacity = 0.0;
+    }
+
+    public void powerOn() {
+        this.isPowerOn = true;
+    }
+
+    public void powerOff() {
+        this.isPowerOn = false;
+    }
+
+    public void fillGas(double gas) {
+        if (this.gas + gas <= this.capacity)
+            this.gas += gas;
+    }
+
+    private double maxDistance() {
+        return gasConsume * gas;
+    }
+
+    public void move(double distance) {
+        this.distance = maxDistance();
+        if (distance < this.distance) {
+            this.distance = distance;
+        }
+        this.gas -= this.distance / this.gasConsume;
+    }
+
+    public double getCapacity() {
+        return capacity;
+    }
+
+    public void setCapacity(double capacity) {
+        this.capacity = capacity;
+    }
+
+    public double getGasConsume() {
+        return gasConsume;
+    }
+
+    public void setGasConsume(double gasConsume) {
+        this.gasConsume = gasConsume;
+    }
+
+    public double travel() {
+        return distance;
+    }
+
+    public double getGas() {
+        return gas;
+    }
+
+    public int comparePerformance(Car car) {
+        return (this.distance < car.distance) ? 1 :
```

Comment:
You need to use your getters for the comparison with the external object, also it would be recommend to return a boolean value instead of an integer

Reply:
fixed